src/test/scala/org/broadinstitute/dsde/agora/server/webservice/PermissionIntegrationSpec.scala

```diff
@@ -246,4 +249,48 @@ class PermissionIntegrationSpec extends FlatSpec with RouteTest with ScalatestRo
       }
   }
 
+  "Agora" should "successfully list permissions for multiple methods simultaneously" in {
+
+    val payload:Seq[AgoraEntity] = Seq(
+      AgoraEntity(agoraEntity1.namespace, agoraEntity1.name, agoraEntity1.snapshotId),
+      AgoraEntity(agoraEntity2.namespace, agoraEntity2.name, agoraEntity2.snapshotId),
+      AgoraEntity(agoraEntity1.namespace, agoraEntity1.name, Some(12345))
+    )
+
+    Post(ApiUtil.Methods.withLeadingVersion + "/permissions", payload) ~>
+      methodsService.multiEntityPermissionsRoute ~>
+      check {
+        assert(status == OK)
+        val entityAclList = responseAs[Seq[EntityAccessControl]]
+        assertResult(3) {entityAclList.size}
+
+        // check first - should get permissions
+        {
```

Comment:
I like the standalone scope braces, I think this is the first time I've seen this in practice